www/staff/index.html

```diff
@@ -76,14 +84,26 @@ <h4>North America</h4>
   <li><a href="/players/Pepe/">Pepe</a></li>
   <li><a href="/players/Skeith/">Skeith</a></li>
   <li><a href="/players/tela/">tela</a></li>
+  <li><a href="/players/Welshi/">Welshi</a></li>
   <li><a href="/players/-124-GH0sT-124-/">|GH0sT|</a></li>
 </ul>
 <h4>South Africa</h4>
 <ul class="staff">
   <li><a href="/players/garry/">garry</a></li>
   <li><a href="/players/Mariac/">Mariac</a></li>
+  <li><a href="/players/XxDarkNezZ/">XxDarkNezZ</a></li>
 </ul>
-<h4>CHN</h4>
+<h4>Asia: Malaysia/Singapore</h4>
+<ul class="staff">
+  <li><a href="/players/cheeser0613/">cheeser0613</a></li>
+  <li><a href="/players/k1r1ha/">k1r1ha</a></li>
+</ul>
+<h4>Asia: Taiwan</h4>
```

Comment:
Why split Asia into two areas?

Reply:
I seperated the countries like we do with china. Maybe we can use a new header to make it look more clean?
The sgp, jp and taiwanese communities play on the same server most of the time, so I wasn't sure what to call the region

Reply:
or did you want to move them into the CHN category? They rarely play there

Reply:
"Asia"

Reply:
> "Asia"

what about korea? also just asia?

Reply:
or is this about their languages? I don't know all the chinese dialects or how much they differ

Reply:
We can just keep it as is.